applications/search-api/src/main/java/no/dcat/portal/query/DatasetsQueryService.java

```diff
@@ -706,4 +713,15 @@ private AggregationBuilder createAggregation(String terms, String field, String
         return new ResponseEntity<>(response.toString(), HttpStatus.OK);
     }
 
+    private String[] includeReturnFields(String returnFields){
```

Comment:
Vi trenger en test som dekker returnfields, med tilfellene "" (tom string) "id" (et felt, som eksisterer i elastic) "id,title" (to felt som eksisterer) og "asdasdasd" (et felt som ikke eksisterer i elastic).

Reply:
felter som ikke finnes blir bare ignorert!